libpod/pod.go

```diff
@@ -70,6 +72,14 @@ type PodConfig struct {
 
 	// ID of the pod's lock
 	LockID uint32 `json:"lockID"`
+
+	//ResourceLimits contains linux specific CPU data for the pod
+	ResourceLimits *spec.LinuxResources `json:"resource_limits,omitempty"`
+
+	// CPU period of the cpuset, determined by --cpus
+	CPUPeriod uint64 `json:"cpu_period,omitempty"`
```

Comment:
Do we need to store these separately? They're already in ResourceLimits

Reply:
I was keeping them around just to be explicit and also helps avoid more handling of nil pointers with a generic podman pod create @mheon

Reply:
How so? It seems like we ought to be able to avoid having the uints if we just nil-check ResourceLimits before accessing it?

Reply:
ok, I will just nil check. Should I keep the getter functions for quota and period?

Reply:
Sure, they can just look at the ResourceLimits bit. Best part, if you put the null checks in there, and use the getters/setters everywhere - you only need to add them once